Tasks/Xcode/task.json

```diff
@@ -67,10 +67,10 @@
         {
             "name": "xcWorkspacePath",
             "type": "filePath",
-            "label": "Workspace Path",
+            "label": "Workspace/Project Path",
             "defaultValue": "**/*.xcodeproj/*.xcworkspace",
             "required": false,
-            "helpMarkDown": "Optional relative path from repo root where xcworkspace folder exists. Scheme must also be specified if value provided. Omit if you intend to use the -target flag under Advanced Arguments."
+            "helpMarkDown": "Optional relative path from repo root where xcworkspace folder exists. Path to the *.xcodeproj can also be specified. Scheme must also be specified if value provided. Omit if you intend to use the -target flag under Advanced Arguments."
```

Comment:
Reading this for the first time in a long time, the whole sentence would make more sense to me as:

Optionally provide a path to a '.xcworkspace' folder relative to the root of the repository. For example: 'src/MyApp.xcworkspace'. Alternatively, to build a specific project in a workspace, provide a path to an '.xcodeproj' file relative to the root of the repository. For example: 'src/MyApp.xcworkspace/MyApp.xcodeproj'. If you provide a value, the Scheme field must also have a value. Leave this field empty if you use the '-target' argument in the Advanced Arguments section.